packages/iTwinUI-react/stories/core/Table.stories.tsx

```diff
@@ -2349,6 +2349,73 @@ Virtualized.argTypes = {
   data: { control: { disable: true } },
 };
 
+export const ScrollToItem: Story<Partial<TableProps>> = (args) => {
+  const onClickHandler = (
+    props: CellProps<{ name: string; description: string }>,
+  ) => action(props.row.original.name)();
+
+  const columns = useMemo(
+    () => [
+      {
+        Header: 'Table',
+        columns: [
+          {
+            id: 'name',
+            Header: 'Name',
+            accessor: 'name',
+          },
+          {
+            id: 'description',
+            Header: 'Description',
+            accessor: 'description',
+          },
+          {
+            id: 'click-me',
+            Header: 'Click',
+            width: 100,
+            Cell: (props: CellProps<{ name: string; description: string }>) => {
+              const onClick = () => onClickHandler(props);
+              return <Anchor onClick={onClick}>Click me!</Anchor>;
+            },
+          },
+        ],
+      },
+    ],
+    [],
+  );
+
+  const data = useMemo(() => {
+    const size = 100000;
+    const arr = new Array(size);
+    for (let i = 0; i < size; ++i) {
+      arr[i] = {
+        id: i,
+        name: `Name${i}`,
+        description: `Description${i}`,
```

Comment:
Minor, but maybe change "Description12345" to "Item scrolled to" or something so that users better understand what's going on in the story?